### DISCUSSION TITLE
interactiv graph

### DISCUSSION THREAD

**Discussion by Douxvalkyn** [qmd-lab/closeread#145] [General]
Hello ,
I just tried an interactiv graph inside a closeread and it doesnt work correctly.

This simple section doesnt work, the plot isn't interactiv:
```
:::{#cr-figa}

```{r}
#| label: figa
#| warning: false

library(ggplot2)
library(plotly)

p <- ggplot(mtcars, aes(x = mpg, y = hp)) +   geom_point()
ggplotly(p)

:::
```

If I remove `#| warning: false` , graph is partially interactiv with `quarto preview` but not at all with `quarto render` :(  and warnings are visible :(

**Comment by andrewpbray** [comment]
Hi @Douxvalkyn !  I'm afraid I'm not able to reproduce. Here is the doc that I've made and I can't detect any issues with it after render.

````markdown
---
title: "Untitled"
format: closeread-html
---

:::{.cr-section}

As you can see in this picture @cr-figa

:::{#cr-figa}

```{r}
#| label: figa
#| warning: false

library(ggplot2)
library(plotly)

p <- ggplot(mtcars, aes(x = mpg, y = hp)) +   geom_point()
ggplotly(p)
```

:::

:::
````

**Comment by jimjam-slam** [comment]
I also couldn't replicate this, @Douxvalkyn :(

https://github.com/user-attachments/assets/ade29cf2-5f07-4112-940e-e30024aba47a

Are you able to give us some information on your setup to help us reproduce the problem?

- R and plotly package versions
- Quarto version
- Closeread version (you can find it in the `_extension.yml` file in your project where the Closeted extension lives)
- Browser and OS

**Comment by Douxvalkyn** [comment]
@jimjam-slam @andrewpbray 
You can see here:
https://douxvalkyn.gitlab.io/closeread/closeread.html

and the code:
https://gitlab.com/douxvalkyn/closeread
